features/step_definitions/steps.rb

```diff
@@ -83,10 +83,6 @@
   raise "STDOUT doesn't contain '#{txt}':\n#{@stdout}" unless @stdout.include?(txt)
 end
 
-Then(/^Stdout is empty$/) do
```

Comment:
@pnatashap, I don't understand why you decided to remove some steps from `steps.rb`. The [report](https://github.com/cqfn/pdd/actions/runs/7736405935/job/21093701790?pr=237#step:5:492) states the following:
```
28 scenarios (23 undefined, 5 passed)
138 steps (34 skipped, 28 undefined, 76 passed)
```
However, we used to have:
```
28 scenarios (28 passed)
138 steps (138 passed)
```
Maybe it's worth keeping all the scenarios passing?

Reply:
Locally it was displayed, that it is duplicated steps and should be removed. Forget that they do not fail. Returned them back

Reply:
Add a flag to cucumber run to fail the run in case of undefined step